packages/hardhat-plugin-ethers/src/ethers-ignition-helper.ts

```diff
@@ -134,10 +135,13 @@ export class EthersIgnitionHelper {
             deploymentId
           );
 
+    const executionEventListener = new PrettyEventHandler();
```

Comment:
The flag would control if this is initialized or not.

Same for the viem plugin.